### DISCUSSION TITLE
Error using resource dictionaries in version 4.7

### DISCUSSION THREAD

**Discussion by mcNets** [unoplatform/uno#11075] [Q&A] [answered]
After upgrading Uno templates and running uno-check to get the last workloads,  I've generated a new UNO app (let call it MyApp) with the new templates. 

I tried to add some resource dictionaries by adding a new Styles folder in MyApp project, and then I've edited App.xaml in the Windows head like this:
```
<Application
    x:Class="X3Produccio.App"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:X3Produccio">
	
    <Application.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
                <!-- Add resource dictionaries here -->
                <ResourceDictionary Source="/Styles/FontSizes.xaml" />
                <ResourceDictionary Source="/Styles/Thickness.xaml" />
                <ResourceDictionary Source="/Styles/TextBlock.xaml" />
                <ResourceDictionary Source="/Styles/X3Produccio.xaml" />
            </ResourceDictionary.MergedDictionaries>
            <!-- Add resources here -->
        </ResourceDictionary>
    </Application.Resources>

</Application>
```

Then running MyApp.Windows in debug mode I'm getting an UnhandledException saying:

>XAML parsing failed
>Cannot locate resource from 'ms-resource:///Files/Styles/FontSizes.xaml'. [Line: 12 Position: 37]

Is there any specific way to use Resource Dictionaries with this new version?

**Comment by jeromelaban** [comment, marked as answer]
As you've placed the resources in the new library, you'll need to use the `ms-appx:///MyApp/Styles/FontSizes.xaml` instead. Did you read this particular setup in a document Uno is providing? If so, it might need an update.

**Reply by mcNets** [reply]
I read the newsletter, it talks about fonts `<TextBlock FontFamily=”ms-appx:///myprojectname/Assets/Fonts/MyFont.ttf” Text=”Text with my font!” />` but not resource dictionaries with styles. Can I find more info about this version, for instance, how to upgrade a previous version?

**Reply by jeromelaban** [reply]
We'll make changes to the documentation to account for this. Thanks!
